src/material/datepicker/calendar-body.spec.ts

```diff
@@ -167,6 +174,32 @@ describe('MatCalendarBody', () => {
       cells = Array.from(fixture.nativeElement.querySelectorAll('.mat-calendar-body-cell'));
     });
 
+    it('should have aria-description on start and end days of a 3-day range', () => {
+      testComponent.startValue = 2;
+      testComponent.endValue = 4;
+      fixture.detectChanges();
+
+      expect(cells[1].hasAttribute('aria-description')).toBe(true);
```

Comment:
I think the `hasAttribute` assertions here are redundant, because if the attribute doesn't exist, the `getAttribute` calls below will return `null` which will fail the assertion.